modules/swagger-models/src/main/java/io/swagger/models/Swagger.java

```diff
@@ -272,6 +272,11 @@ public void addProduces(String produces) {
         return sorted;
     }
 
+    @JsonIgnore
+    public Map<String, Path> getPathsMap() {
```

Comment:
Yes, I would suggest removing it and opening a new PR to _remove_ the sorting on the existing method.  I see no reason why we need to monkey with the sort order when fetching the paths.